test/content.test.js

```diff
@@ -47,6 +47,26 @@ describe('Profile', () => {
       expect(profileSet.destProfiles[2]).to.deep.include({ profile: 'target2', role_name: 'DefaultRole' });
       expect(profileSet.excludedNames[0]).to.eq('target4');
     })
+
+    it('loads the configuration that contains base account with target_region', () => {
+      let profileSet = new ProfileSet([
+        { profile: 'target1', aws_account_id: '111122223334',
+          role_name: 'Role1', source_profile: 'base1' },
+        { profile: 'target2', aws_account_id: '111122223335',
+          source_profile: 'base1' },
```

Comment:
Please set the individual region for the test to overwrite the default one

Reply:
Done.